go/vt/vttablet/tabletmanager/vreplication/vreplicator.go

```diff
@@ -55,7 +55,8 @@ var (
 	// to ensure that it satisfies liveness criteria implicitly expected by internal processes like Online DDL
 	vreplicationMinimumHeartbeatUpdateInterval = 60
 
-	vreplicationExperimentalFlagOptimizeInserts int64 = 1
+	vreplicationExperimentalFlagOptimizeInserts    int64 = 1
+	vreplicationExperimentalParallelizeBulkInserts int64 = 0x02
```

Comment:
Any reason not to use the same base as the variable, i.e. 2?

Reply:
I'm not sure, I think I haven't fully understood how flags are intended to work. In any case, will be following your suggestion above not to use experimental flags at all for this PR.